app/views/users/new.html.erb

```diff
@@ -1 +1,19 @@
-<% provide(:title, t(".signup")) %>
+<% provide :title, t(".signup") %>
+<h1><%= t ".signup" %></h1>
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for(@user) do |f| %>
+      <%= render "shared/error_messages" %>
+      <%= f.label :name, t(".name") %>
```

Comment:
ko nên khai báo tên label thủ công như hiện tại, dùng theo kiểu này và khai báo trong i18n để tránh khai báo nhiều lần ở nhiều form nha em, tham khảo https://guides.rubyonrails.org/i18n.html#translations-for-active-record-models
dùng như này thôi:
```suggestion
      <%= f.label :name %>
```